internal/cli/cli.go

```diff
@@ -29,23 +29,19 @@ func configuration() {
 	addEntry("Version", config.DiceDBVersion)
 
 	// Add the port number on which DiceDB is running to the configuration table
-	addEntry("Port", config.DiceConfig.AsyncServer.Port)
+	addEntry("Port", config.DiceConfig.RespServer.Port)
 
 	// Add whether multi-threading is enabled to the configuration table
-	addEntry("Multi Threading Enabled", config.DiceConfig.Performance.EnableMultiThreading)
+	addEntry("Multi Threading Enabled", true)
```

Comment:
Do we need to render this now? As we are multithreaded for all cases

Reply:
For the initial few days, I thought of keeping it just so as not to confuse users. But I'm okay with anything.

Reply:
Yes, I think mentioning multithreading on startup for now makes sense